index.html

```diff
@@ -1052,6 +1100,73 @@
 
 </div>
 
+## Collective Privacy {#collective-privacy}
+
+<div class="practice">
+
+<p>
+  <span class="practicelab" id="principle-collective-privacy">
+    Groups and various forms of institutions may best protect and support autonomy by making
+    decisions that apply collectively rather than individually to either prevent or enable data
```

Comment:
They should probably make the decisions collectively, rather than just making them apply collectively?

Reply:
Yes! Fixed.